cranelift/assembler-x64/meta/src/dsl.rs

```diff
@@ -4,10 +4,13 @@
 //! - compact--i.e., define an x64 instruction on a single line, and
 //! - a close-to-direct mapping of what we read in the x64 reference manual.
 
+pub(crate) mod custom;
+pub(crate) mod display;
```

Comment:
```suggestion
```

We don't want any implementation of the actual custom functions to live in this crate; we want them to live in the parent crate, `cranelift-assembler-x64`, in the `custom` module. So that `custom` module, which you modified to contain a `visit` sub-module, should also contain a `display` sub-module and eventually an `encode` module, once we need it.